src/Nri/Ui/SegmentedControl/V11.elm

```diff
@@ -60,14 +60,14 @@ not a page control
   - `width`: how to size the segmented control
 
 -}
-viewSelect :
+viewRadioGroup :
```

Comment:
We checked the monorepo, and we're often using the `viewSelect` helper when it would be better to use `view`. I want users of this function to know what they're getting -- if they wouldn't use radio buttons, they shouldn't user viewRadioGroup